mrp_operations_time_control/models/operation_time.py

```diff
@@ -33,7 +33,8 @@ def _create_operation_line(self):
             'user': self.env.uid})
 
     def _write_end_date_operation_line(self):
-        self.operation_time_lines[-1].end_date = fields.Datetime.now()
+        if self.operation_time_lines:
+            self.operation_time_lines[-1].end_date = fields.Datetime.now()
```

Comment:
And what about `[-1:]` I see you are using it in the other modified file (not sure if it has to be in this PR)